build/node-feature-discovery/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.13.5-alpine as builder
+
+ARG VERSION=v0.6.0
+ARG TARGETPLATFORM
+ARG HOSTMOUNT_PREFIX
```

Comment:
I don't believe we're setting the --build-arg for this in our command, we might be best to just set the default to what they're using upstream in their Makefile https://github.com/kubernetes-sigs/node-feature-discovery/blob/c2503a20abdda262ca33d4c81d23d14b452ab8c9/Makefile#L19